runtime/altair/src/lib.rs

```diff
@@ -2241,6 +2241,15 @@ impl_runtime_apis! {
 		}
 	}
 
+	// PoolFeesApi
+	impl runtime_common::apis::PoolFeesApi<Block, PoolId, PoolFeeId, AccountId, Balance, Rate> for Runtime {
+		fn list_fees(pool_id: PoolId) -> Option<Vec<(cfg_traits::fee::PoolFeeBucket, Vec<cfg_types::pools::PoolFee<AccountId, PoolFeeId, cfg_types::pools::PoolFeeAmounts<Balance, Rate>>>)>> {
```

Comment:
NIT. This type is very long. Could maybe `get_pool_fees()` return a type which simplifies this?

Reply:
Since clippy is complaining, I should do that yes. I dislike introducing types specifically for runtime API responses because these have to be typed manually in the JS side.

Reply:
Oh, I didn't know that. And with tuples, they do not need to specify any type?

Reply:
Maybe you can get the best of both worlds with a type alias?

Reply:
In JS, tuples are just arrays. So you have to specify them as tuples still. WDYT about [731aaa1](https://github.com/centrifuge/centrifuge-chain/pull/1728/commits/731aaa1d7fd7e59ad05c9a32c196cbd8d09714ec)?

Reply:
Perfect!